docs/spec/spec.md

```diff
@@ -151,13 +163,14 @@ The database schema will contain the code to create,
 Apart from the `bal build` users can also use the following command to generate the same as above mentioned,
 
 ```bash
-bal persist generate --datastore mysql --module db
+bal persist generate --datastore mysql --module db --test-datastore="test_datastore"
```

Comment:
Isn't it better to have `--test-datastore` with either `h2` or `inmemory`, because other options have real values used in real-world scenarios? EX: In `--datastore` we have specified `mysql`.

`bal persist generate --datastore mysql --module db --test-datastore h2`